### PR TITLE
Transcation system

### PR DESCRIPTION
**Pull request by badb**

### CODE REVIEW

File: gnr/customizers/configurationdialogcustomizer.py [hunk 1]
```diff
@@ -328,6 +336,14 @@ def __recount_blender_performance(self):
 
     def __read_payment_config(self, cfg_desc):
         cfg_desc.eth_account = u"{}".format(self.gui.ui.ethAccountLineEdit.text())
+        try:
+            cfg_desc.min_price = float(self.gui.ui.minPriceLineEdit.text())
+        except ValueError as err:
+            logger.warning("Wrong min_payment value: {}".format(err))
```
Comment (chfast) [inline gnr/customizers/configurationdialogcustomizer.py:342]: Does it mean invalid values will not be report in the UI and will not affect the client?

File: gnr/customizers/newtaskdialogcustomizer.py [hunk 2]
```diff
@@ -37,10 +39,11 @@ def _setup_connections(self):
         self._setup_basic_new_task_connections()
         self._setup_advance_new_task_connections()
         self._setup_options_connections()
+        self._setup_payment_connections()
 
     def _setup_task_type_connections(self):
-        QtCore.QObject.connect(self.gui.ui.taskTypeComboBox, QtCore.SIGNAL("currentIndexChanged(const QString)"),
-                               self._task_type_value_changed)
+        QObject.connect(self.gui.ui.taskTypeComboBox, SIGNAL("currentIndexChanged(const QString)"),
+                        self._task_type_value_changed)
```
Comment (chfast) [inline gnr/customizers/newtaskdialogcustomizer.py]: Have you tried new syntax:

```
self.gui.ui.taskTypeComboBox.currentIndexChanged.connect(self._task_type_value_changed)
```

Also: shouldn't it be `"currentIndexChanged(const QString&)"?

File: gnr/customizers/newtaskdialogcustomizer.py [hunk 3]
```diff
@@ -50,22 +53,39 @@ def _setup_basic_new_task_connections(self):
         self.gui.ui.cancelButton.clicked.connect(self._cancel_button_clicked)
 
     def _setup_advance_new_task_connections(self):
-        QtCore.QObject.connect(self.gui.ui.optimizeTotalCheckBox, QtCore.SIGNAL("stateChanged(int) "),
-                               self._optimize_total_check_box_changed)
+        QObject.connect(self.gui.ui.optimizeTotalCheckBox, SIGNAL("stateChanged(int) "),
```
Comment (chfast) [inline gnr/customizers/newtaskdialogcustomizer.py]: You have tailing space here.

File: gnr/customizers/configurationdialogcustomizer.py (hunk 1 above)
Comment (badb) [inline gnr/customizers/configurationdialogcustomizer.py:342]: Yes, but it's a greater issue. We should do proper warnings for other configuration variables. I put it in separated issue: https://github.com/imapp-pl/golem/issues/213 and i'll try to address it later.

**Pull request merged**